ci/build-mingw64.sh

```diff
@@ -199,27 +261,33 @@ fi
 CFLAGS+=" -I'$prefix_dir/include'"
 LDFLAGS+=" -L'$prefix_dir/lib'"
 export CFLAGS LDFLAGS
-rm -rf build
+build=mingw_build
+rm -rf $build
 
-meson setup build --cross-file "$prefix_dir/crossfile" \
+meson setup $build --cross-file "$prefix_dir/crossfile" \
     --buildtype debugoptimized \
     -Dlibmpv=true -Dlua=luajit \
     -D{shaderc,spirv-cross,d3d11,libplacebo}=enabled
 
-meson compile -C build
+meson compile -C $build
 
 if [ "$2" = pack ]; then
     mkdir -p artifact
     echo "Copying:"
-    cp -pv build/player/mpv.com build/mpv.exe "$prefix_dir/bin/"*.dll artifact/
-    # ship everything and the kitchen sink
+    cp -pv $build/player/mpv.com $build/mpv.exe artifact/
+    # everything and the kitchen sink
+    cp -pv "$prefix_dir/bin/"*.dll artifact/
     shopt -s nullglob
     for file in /usr/lib/gcc/$TARGET/*-posix/*.dll /usr/$TARGET/lib/*.dll; do
         cp -pv "$file" artifact/
     done
+    # delete unneeded DLLs
+    for file in artifact/lib{gomp,gfortran,objc,quadmath,atomic,harfbuzz-subset,SPIRV-Tools,charset}*.dll; do
+        rm -v "$file"
+    done
+
     echo "Archiving:"
     pushd artifact
-    wine64 ./mpv.com -v --no-config
     zip -9r "../mpv-git-$(date +%F)-$(git rev-parse --short HEAD)-${TARGET%%-*}.zip" -- *
```

Comment:
While at it we could improve in this area. Do not double zip and upload libmpv too. Probably as separate package.

Reply:
Is there an easy way? AFAIK upload-artifact can't upload bare files (it always zips) and it would be annoying to move the zipping there

Reply:
There seems to be no other way, than feeding list of files to `upload-artifact` directly.